modules/cli/apply/apply.go

```diff
@@ -181,48 +181,42 @@ func save(client *client.Getter, bundle *fleet.Bundle, imageScans ...*fleet.Imag
 
 	obj, err := c.Fleet.Bundle().Get(bundle.Namespace, bundle.Name, metav1.GetOptions{})
 	if apierrors.IsNotFound(err) {
-		_, err = c.Fleet.Bundle().Create(bundle)
-		if err == nil {
-			fmt.Printf("%s/%s\n", bundle.Namespace, bundle.Name)
-		} else {
+		if _, err = c.Fleet.Bundle().Create(bundle); err != nil {
 			return err
 		}
+		fmt.Printf("created: %s/%s\n", bundle.Namespace, bundle.Name)
 	} else if err != nil {
 		return err
 	}
 
 	obj.Spec = bundle.Spec
 	obj.Annotations = mergeMap(obj.Annotations, bundle.Annotations)
 	obj.Labels = mergeMap(obj.Labels, bundle.Labels)
-	_, err = c.Fleet.Bundle().Update(obj)
-	if err == nil {
-		fmt.Printf("%s/%s\n", obj.Namespace, obj.Name)
+	if _, err := c.Fleet.Bundle().Update(obj); err != nil {
+		return err
 	}
+	fmt.Printf("updated: %s/%s\n", obj.Namespace, obj.Name)
```

Comment:
Reviewers: this update was the culprit for https://github.com/rancher/fleet/issues/437